ui/src/ducks/app/monitoring.js

```diff
@@ -20,17 +22,18 @@ const defaultState = {
     list: [],
     error: null,
     isLoading: false,
-    isRefreshing: false
+    isRefreshing: false,
   },
   cluster: {
     apiServerStatus: 0,
     kubeSchedulerStatus: 0,
     kubeControllerManagerStatus: 0,
     error: null,
+    prometheusPodStatusMessage: null,
     isLoading: false,
-    isRefreshing: false
+    isRefreshing: false,
```

Comment:
What's the difference between `loading` and `refreshing`? Confuses me, though not related to this PR.